modules/installation-creating-gcp-security.adoc

```diff
@@ -48,40 +58,97 @@ resources:
 - name: cluster-security
```

Comment:
The firewall rules are hosted in host project and IAM is hosted in service account. And hence we need to separate their creation. The firewall rules are also different between external cluster and internal cluster. The full steps are like these:
For external cluster, create firewall rules:
Get 03_firewall.py from https://github.com/openshift/installer/blob/master/upi/gcp/03_firewall.py

```javascript
cat <<EOF >${DIR}/03_firewall.yaml
imports:
- path: ${DIR}/03_firewall.py
resources:
- name: cluster-firewall
  type: ${DIR}/03_firewall.py
  properties:
    allowed_external_cidr: '0.0.0.0/0'
    infra_id: '${INFRA_ID}'
    cluster_network: '${CLUSTER_NETWORK}'
    network_cidr: '${NETWORK_CIDR}'
EOF
gcloud deployment-manager deployments create ${INFRA_ID}-firewall --config ${DIR}/03_firewall.yaml --project ${HOST_PROJECT} --account ${HOST_PROJECT_ACCOUNT}
```
For internal cluster, create firewall rules:

```javascript
cat <<EOF >${DIR}/03_firewall.yaml
imports:
- path: ${DIR}/03_firewall.py
resources:
- name: cluster-firewall
  type: ${DIR}/03_firewall.py
  properties:
    allowed_external_cidr: '${NETWORK_CIDR}'
    infra_id: '${INFRA_ID}'
    cluster_network: '${CLUSTER_NETWORK}'
    network_cidr: '${NETWORK_CIDR}'
EOF
gcloud deployment-manager deployments create ${INFRA_ID}-firewall --config ${DIR}/03_firewall.yaml --project ${HOST_PROJECT} --account ${HOST_PROJECT_ACCOUNT}
```

To create IAM, get 03_iam.py from https://github.com/openshift/installer/blob/master/upi/gcp/03_iam.py

```javascript
cat <<EOF >${DIR}/03_iam.yaml
imports:
- path: ${DIR}/03_iam.py
resources:
- name: cluster-iam
  type: ${DIR}/03_iam.py
  properties:
    infra_id: '${INFRA_ID}'
EOF
gcloud deployment-manager deployments create ${INFRA_ID}-iam --config ${DIR}/03_iam.yaml
```